modules/core/shared/src/main/scala/lucuma/core/enum/GuideProbe.scala

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma
+package core
+package enum
+
+import cats.syntax.eq._
+import lucuma.core.util.Enumerated
+
+/**
+ * Enumerated type for Guiding probes
+ * @group Enumerations (Generated)
+ */
+sealed abstract class GuideProbe(val tag: String) extends Product with Serializable
```

Comment:
There are more like categories of guide probes rather than particular guide probes. Maybe `GuideProbeCategory` or something?  I can't remember why we need this.  What do we do with it?

Reply:
This is orthogonal to the GAIA bands, correct?

Reply:
AGS indexes the search params by guide probe type. I'm making a few non-trivial changes to ags on how the searches are conducted, maybe I could keep this enum in AGS until I'm sure we are going to need it